examples/two_dimensional_turbulence_shallow_water.jl

```diff
@@ -0,0 +1,112 @@
+using Oceananigans.Architectures: CPU, architecture
+using Oceananigans.Models: ShallowWaterModel
+using Oceananigans.Grids: Periodic, Bounded, RegularCartesianGrid
+using Oceananigans.Grids: xnodes, ynodes, interior
+using Oceananigans.Simulations: Simulation, set!, run!, TimeStepWizard
+using Oceananigans.Coriolis: FPlane
+using Oceananigans.Advection: WENO5
+using Oceananigans.OutputWriters: JLD2OutputWriter, IterationInterval
+using Oceananigans.Fields, Oceananigans.AbstractOperations
+
+using Statistics
+using Plots
+using Printf
+using JLD2
+using LinearAlgebra
+
+Lx = 2π 
+Ly = 2π
+Lz = 2π
+
+Nx = 128
+Ny = Nx
+
+f = 100         
+g = 1
+
+grid = RegularCartesianGrid(
+        size=(Nx, Ny, 1), 
+      extent=(Lx, Ly, Lz),
+    topology=(Periodic, Periodic, Bounded)
+    )
+
+model = ShallowWaterModel(
+                  architecture=CPU(),
+                     advection=WENO5(),
+                          grid=grid,
+    gravitational_acceleration=g,
+                      coriolis=FPlane(f=f)
+    )
+
+hᵢ = model.grid.Lz .+ 0.1*rand(size(model.grid)...)
+
+set!(model, h = hᵢ)
+
+uh, vh, h = model.solution
+
+u_op   = model.solution.uh / model.solution.h
+v_op   = model.solution.vh / model.solution.h
+η_op   = model.solution.h - model.grid.Lz
+ω_op   = @at (Center, Center, Center) ∂x(v_op) - ∂y(u_op)
+s_op   = sqrt(u_op^2 + v_op^2) 
+
+u_field = ComputedField(u_op)
+v_field = ComputedField(v_op)
+η_field = ComputedField(η_op)
+ω_field = ComputedField(ω_op)
+s_field = ComputedField(s_op)
```

Comment:
call this `speed_field`? (similarly `speed_op`, etc?)

Reply:
Also a good suggestion.  Done